bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,323 @@
+        """
+        if amount > 25:
+            await ctx.send("You can't get more than top 25 games.")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 top game.")
+            return
+
+        games = await self.get_games_list(self.http_session, amount, sort='total_rating desc',
+                                          additional_body="where total_rating >= 90; sort total_rating_count desc;")
+
+        pages = await self.get_pages(games)
+        await ImagePaginator.paginate(pages, ctx, Embed(title=f'Top {amount} Games'))
+
+    @games.command(name='genres', aliases=['genre', 'g'])
+    async def genres(self, ctx: Context) -> None:
+        """Get all available genres."""
+        await ctx.send(f"Currently available genres: {', '.join(f'`{genre}`' for genre in self.genres)}")
+
+    @games.command(name='search', aliases=['s'])
+    async def search(self, ctx: Context, *, search: str) -> None:
+        """Find games by name."""
+        lines = await self.search_games(self.http_session, search)
+
+        await LinePaginator.paginate((line for line in lines), ctx, Embed(title=f'Game Search Results: {search}'))
+
+    @games.command(name='company', aliases=['companies'])
+    async def company(self, ctx: Context, amount: int = 5) -> None:
+        """
+        Get random Game Companies companies from IGDB API.
+
+        Support amount parameter. Max is 25, min is 1.
+        """
+        if amount > 25:
+            await ctx.send("You can't get more than 25 companies at once.")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 company.")
+            return
+
+        companies = await self.get_companies_list(self.http_session, amount, random.randint(0, 150))
+        pages = [await self.create_company_page(co) for co in companies]
+
+        await ImagePaginator.paginate(pages, ctx, Embed(title='Random Game Companies'))
+
+    async def get_games_list(self,
+                             client: ClientSession,
+                             limit: int,
+                             genre: str = None,
+                             sort: str = None,
+                             additional_body: str = "",
+                             offset: int = 0) \
+            -> List[Dict[str, Any]]:
+        """Get Games List from IGDB API."""
+        # Create body of IGDB API request, define fields, sorting, offset, limit and genre
+        body = "fields cover.image_id, first_release_date, total_rating, name, storyline, url, platforms.name, "
+        body += "status, involved_companies.company.name, summary, age_ratings.category, age_ratings.rating, "
+        body += "total_rating_count;\n"
+
+        body += f"sort {sort};\n" if sort else ''
+        body += f"offset {offset};\n"
+        body += f"limit {limit};\n"
+
+        body += f"where genres = ({genre});" if genre else ''
+        body += additional_body
```

Comment:
I find the repeated concatenation unnecessary. A string that won't fit into one line can be constructed as such:
```python
body = (
    "a very long "
    "message."
)
```
This won't distract from the content. Using f-strings remains allowed. If the conditionals are necessary, it may be better to assign the desired values to variables with descriptive names, and then constructing the string from them. Since this is an API query, you may want to consider using a template string at the top of the module, and then simply passing in these optional parameters to it as necessary. It's probably ok to keep it here though.